package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bufferapp/components",
-  "version": "0.1.8",
+  "version": "0.1.8-beta-1",
```

Comment:
Love the idea here using a beta here @alvarezmelissa87!

If you wanted to you could publish this under a custom flag (does not push to production/default npm package) with:

```
npm publish --tag beta
```